config/routes.rb

```diff
@@ -7,5 +7,16 @@
       resources :likes
     end
   end
+
+  namespace :api do
+    namespace :v1 do
+      resources :users do
+        resources :posts do
+         resources :comments, only: [:create]
+        end
```

Comment:
- [ ] Kindly add an endpoint to get all comments. What you have is presently the creation of comments.
### Requirements:
![image](https://github.com/MesakDuduCoder/rails_blog_app/assets/50990925/7e38bc3f-aed2-4872-bf2a-b5ca5647adec)

### Endpoints::
![image](https://github.com/MesakDuduCoder/rails_blog_app/assets/50990925/469bddb8-8f43-4cc8-8663-02d2c0b5907c)